test/message-input.test.js

```diff
@@ -21,4 +25,38 @@ describe('message-input', () => {
   it('text area placeholder should be set', () => {
     expect(messageInput.shadowRoot.querySelector('vaadin-text-area').placeholder).to.be.equal('Message');
   });
+
+  describe('submit functionality', () => {
```

Comment:
Missing tests for:
- the field is cleared after submit
- the field is focused after submit
- `event.detail` contains the value
- submit not triggered if the field is empty

Reply:
Done.